### PR TITLE
replaced static text with dynamic in frontend/

### PR DESCRIPTION
**Pull request by feigenmann**

### CODE REVIEW

File: frontend/public/index.html [hunk 1]
```diff
@@ -11,7 +11,7 @@
   </head>
   <body>
     <noscript>
-      <strong>Sorry, eCamp funktioniert nicht ohne JavaScript. Bitte aktiviere JavaScript um fortzufahren.</strong>
+      <strong></strong>
```
Comment (carlobeltrame) [inline frontend/public/index.html]: Bitte wiederherstellen
```suggestion
      <strong>Sorry, eCamp funktioniert nicht ohne JavaScript. Bitte aktiviere JavaScript um fortzufahren.</strong>
```

File: frontend/src/components/camp/CollaboratorListItem.vue [hunk 2]
```diff
@@ -5,26 +5,36 @@
...
+      ]"
+      item-value="key"
+      item-text="translation"
       :my="0"
       dense
       vee-rules="required" />
     <v-list-item-action class="ml-2">
       <button-delete @click="api.del(collaborator)">
-        Remove
+        {{ $tc("components.camp.collaborators.remove") }}
```
Comment (carlobeltrame) [inline frontend/src/components/camp/CollaboratorListItem.vue]: Translation key should be `components.camp.collaboratorListItem.remove` (also remember to change it in the JSON)

File: frontend/src/components/camp/CampPrint.vue [hunk 3]
```diff
@@ -5,16 +5,15 @@
       <v-btn color="primary" class="mt-5"
              :href="previewUrl"
              target="_blank">
-        Open print preview
+        {{ $tc('entity.camp.print.openPrintPreview') }}
```
Comment (carlobeltrame) [inline frontend/src/components/camp/CampPrint.vue]: "Open print preview" is not a field of a camp like "title" or "motto". So this key should be `components.camp.campPrint.openPrintPreview` (also remember to change it in the JSON)

File: frontend/src/components/camp/CampPrint.vue [hunk 4]
```diff
@@ -5,16 +5,15 @@
...
       </v-btn>
       <v-btn
         color="primary"
         class="mt-5 ml-5"
         :loading="printing"
         @click="print">
-        Print now
+        {{ $tc('entity.camp.print.printNow') }}
```
Comment (carlobeltrame) [inline frontend/src/components/camp/CampPrint.vue]: Same here, should be `components.camp.campPrint.printNow` (also remember to change it in the JSON)

File: frontend/src/components/form/base/EColorPicker.vue [hunk 5]
```diff
@@ -34,7 +34,7 @@ export default {
   name: 'ColorPicker',
   components: { BasePicker },
   props: {
-    value: { type: String, required: true }
+    value: { type: Object, required: true }
```
Comment (carlobeltrame) [inline frontend/src/components/form/base/EColorPicker.vue:39]: Revert, change does not belong in this PR
```suggestion
    value: { type: String, required: true }
```

File: frontend/src/components/layout/AuthContainer.vue [hunk 6]
```diff
@@ -11,7 +11,7 @@
       </v-card>
     </v-col>
     <span v-if="$vuetify.breakpoint.smAndUp" class="photo_credits ma-3 pa-1">
-      Photo by Markus Rohner / Lotos
+      {{ $tc('views.auth.container.photoCredits') }}
```
Comment (carlobeltrame) [inline frontend/src/components/layout/AuthContainer.vue]: Key should be `components.layout.authContainer.photoCredits` (also remember to change it in the JSON)

File: frontend/src/views/NavigationDefault.vue [hunk 7]
```diff
@@ -12,17 +12,17 @@
     v-else app
     fixed grow>
     <v-btn :to="{name: 'home'}">
-      <span>Home</span>
+      <span>{{ $tc('components.navigation.home') }}</span>
```
Comment (carlobeltrame) [inline frontend/src/views/NavigationDefault.vue]: Key should be `views.navigationDefault.home` (also remember to change it in the JSON)

File: frontend/src/views/NavigationDefault.vue [hunk 8]
```diff
@@ -12,17 +12,17 @@
...
       <span>
         <v-icon>$vuetify.icons.ecamp</v-icon>️
       </span>
     </v-btn>
     <v-btn :to="{name: 'camps'}">
-      <span>Meine Camps</span>
+      <span>{{ $tc('components.navigation.myCamps') }}</span>
```
Comment (carlobeltrame) [inline frontend/src/views/NavigationDefault.vue]: Key should be `views.navigationDefault.myCamps` (also remember to change it in the JSON)

File: frontend/src/views/NavigationDefault.vue [hunk 9]
```diff
@@ -12,17 +12,17 @@
...
       <v-icon>mdi-format-list-bulleted-triangle</v-icon>
     </v-btn>
     <v-btn :to="{name: 'profile'}">
-      <span>Profile</span>
+      <span>{{ $tc('components.navigation.profile') }}</span>
```
Comment (carlobeltrame) [inline frontend/src/views/NavigationDefault.vue]: Key should be `views.navigationDefault.profile` (also remember to change it in the JSON)

File: frontend/src/views/auth/LoginCallback.vue [hunk 10]
```diff
@@ -3,7 +3,7 @@
     <v-container class="fill-height" fluid>
       <v-row align="center" justify="center">
         <v-card>
-          <v-card-text><h3>Logging you in</h3></v-card-text>
+          <v-card-text><h3>{{$tc('auth.login.loginInProgress')</h3></v-card-text>
```
Comment (carlobeltrame) [inline frontend/src/views/auth/LoginCallback.vue]: Key should be `views.auth.loginCallback.loginInProgress` (also remember to change it in the JSON)

File: frontend/src/views/auth/RegisterDone.vue [hunk 11]
```diff
@@ -1,14 +1,14 @@
 <template>
   <auth-container>
-    <h1 class="display-1">Konto erstellen…</h1>
+    <h1 class="display-1">{{ $tc('auth.register.title') }}</h1>
```
Comment (carlobeltrame) [inline frontend/src/views/auth/RegisterDone.vue:3]: Key should be `auth.registerDone.title` (should probably be a new translation key in the JSON)

File: frontend/src/views/auth/RegisterDone.vue [hunk 12]
```diff
@@ -1,14 +1,14 @@
...
     <v-alert type="success" class="my-4 text--green text--darken-2">
-      Successfully registered.
+     {{ $tc('auth.register.success') }}
```
Comment (carlobeltrame) [inline frontend/src/views/auth/RegisterDone.vue]: Key should be `auth.registerDone.success` (also remember to change it in the JSON)

File: frontend/src/views/auth/RegisterDone.vue [hunk 13]
```diff
@@ -1,14 +1,14 @@
...
     </v-alert>
     <v-spacer />
     <v-btn color="primary" :to="{name: 'login'}"
            x-large
            class="my-4" block>
-      Login
+       {{ $tc('auth.register.login') }}
```
Comment (carlobeltrame) [inline frontend/src/views/auth/RegisterDone.vue]: Key should be `auth.registerDone.login` (also remember to change it in the JSON)

File: frontend/src/views/camp/NavigationCamp.vue [hunk 14]
```diff
@@ -7,23 +7,23 @@
              text
              width="216">
         <v-toolbar-title>
-          {{ camp().title | loading('Camp wird geladen…') }}
+          {{ camp().title | loading('Lager wird geladen…') }}
```
Comment (carlobeltrame) [inline frontend/src/views/camp/NavigationCamp.vue]: You can use `$tc` in here to translate this as well:
`loading($tc('some.translation.key'))`

File: frontend/src/views/camp/NavigationCamp.vue [hunk 15]
```diff
@@ -7,23 +7,23 @@
...
         </v-toolbar-title>
       </v-btn>
     </logo>
 
     <v-toolbar-items>
       <v-btn :to="campRoute(camp(), 'collaborators')" text>
         <v-icon :left="$vuetify.breakpoint.mdAndUp">mdi-account-group</v-icon>
-        <span class="sr-only-sm-and-down">Team</span>
+        <span class="sr-only-sm-and-down">{{ $tc('components.navigation.camp.team') }}</span>
```
Comment (carlobeltrame) [inline frontend/src/views/camp/NavigationCamp.vue]: Key should be `views.camp.navigationCamp.team`

File: frontend/src/views/camp/NavigationCamp.vue [hunk 16]
```diff
@@ -7,23 +7,23 @@
...
       </v-btn>
       <v-btn :to="campRoute(camp(), 'admin')" text>
         <v-icon :left="$vuetify.breakpoint.mdAndUp">mdi-cogs</v-icon>
-        <span class="sr-only-sm-and-down">Admin</span>
+        <span class="sr-only-sm-and-down">{{ $tc('components.navigation.camp.admin') }}</span>
```
Comment (carlobeltrame) [inline frontend/src/views/camp/NavigationCamp.vue]: Key should be `views.camp.navigationCamp`

File: frontend/src/views/camp/NavigationCamp.vue [hunk 17]
```diff
@@ -7,23 +7,23 @@
...
       </v-btn>
       <v-btn :to="campRoute(camp(), 'print')" text>
         <v-icon :left="$vuetify.breakpoint.mdAndUp">mdi-printer</v-icon>
-        <span class="sr-only-sm-and-down">Print</span>
+        <span class="sr-only-sm-and-down">{{ $tc('components.navigation.camp.admin') }}</span>
```
Comment (carlobeltrame) [inline frontend/src/views/camp/NavigationCamp.vue]: Key should be `views.camp.navigationCamp.admin`

File: frontend/src/views/camp/NavigationCamp.vue [hunk 18]
```diff
@@ -33,23 +33,23 @@
   <v-bottom-navigation v-else app
                        fixed grow>
     <v-btn>
-      <span>Material</span>
+      <span>{{ $tc('components.navigation.camp.material') }}</span>
```
Comment (carlobeltrame) [inline frontend/src/views/camp/NavigationCamp.vue]: Same here

File: frontend/src/views/camp/NavigationCamp.vue [hunk 19]
```diff
@@ -33,23 +33,23 @@
...
       <v-icon>mdi-package-variant</v-icon>
     </v-btn>
     <v-btn>
-      <span>Tasks</span>
+      <span>{{ $tc('components.navigation.camp.tasks') }}</span>
```
Comment (carlobeltrame) [inline frontend/src/views/camp/NavigationCamp.vue]: Same here

File: frontend/src/views/camp/NavigationCamp.vue [hunk 20]
```diff
@@ -33,23 +33,23 @@
...
       <v-icon>mdi-format-list-checks</v-icon>
     </v-btn>
     <v-btn :to="{name: 'camp/program'}">
-      <span>Camp</span>
+      <span>{{ $tc('components.navigation.camp.camp') }}</span>
```
Comment (carlobeltrame) [inline frontend/src/views/camp/NavigationCamp.vue]: Same here

File: frontend/src/views/camp/NavigationCamp.vue [hunk 21]
```diff
@@ -33,23 +33,23 @@
...
       <v-icon large>mdi-tent</v-icon>
     </v-btn>
     <v-btn :to="{name: 'camp/collaborators'}" exact>
-      <span>Team</span>
+      <span>{{ $tc('components.navigation.camp.team') }}</span>
```
Comment (carlobeltrame) [inline frontend/src/views/camp/NavigationCamp.vue]: Same here

File: frontend/src/views/camp/NavigationCamp.vue [hunk 22]
```diff
@@ -33,23 +33,23 @@
...
       <v-icon>mdi-account-group</v-icon>
     </v-btn>
     <v-btn :to="{name: 'camp/admin'}">
-      <span>Admin</span>
+      <span>{{ $tc('components.navigation.camp.admin') }}</span>
```
Comment (carlobeltrame) [inline frontend/src/views/camp/NavigationCamp.vue]: Same here

File: frontend/src/views/camp/Print.vue [hunk 23]
```diff
@@ -3,7 +3,7 @@ Admin screen of a camp: Displays details & periods of a single camp and allows t
 -->
 
 <template>
-  <content-card title="Print">
+  <content-card :title="$tc('navigation.titles.print')">
```
Comment (carlobeltrame) [inline frontend/src/views/camp/Print.vue]: Key should be `views.camp.print.title`

**Review by carlobeltrame — CHANGES_REQUESTED** [review]
Translation keys should match the file and directory names. All (or most) translations that are used in a file `frontend/src/components/foo/Bar.vue` should look like `components.foo.bar.something`

File: frontend/src/components/camp/CampPrint.vue (hunk 3 above)
Comment (feigenmann) [reply frontend/src/components/camp/CampPrint.vue]: done

File: frontend/src/components/layout/AuthContainer.vue (hunk 6 above)
Comment (feigenmann) [reply frontend/src/components/layout/AuthContainer.vue]: han noch eine typo bei mir könenn swir zsaummen anschauen

File: frontend/src/views/camp/NavigationCamp.vue (hunk 14 above)
Comment (feigenmann) [reply frontend/src/views/camp/NavigationCamp.vue]: added auth.loginCallback.CampIsLoading

File: frontend/src/views/camp/NavigationCamp.vue (hunk 15 above)
Comment (feigenmann) [reply frontend/src/views/camp/NavigationCamp.vue]: typo

**Review by feigenmann — COMMENTED** [review]
Done

File: .vscode/settings.json [hunk 24]
```diff
@@ -4,4 +4,8 @@
     "eslint.format.enable": true,
     "phpunit.files": "backend/{module,content-type}/**/test/**/*Test.php",
     "phpunit.args": [ "-cbackend/phpunit.xml" ],
-}
\ No newline at end of file
+  "editor.codeActionsOnSave": {
+    "source.fixAll.eslint": true
+  },
+  "eslint.validate": ["javascript"]
+}
```
Comment (carlobeltrame) [inline .vscode/settings.json:11]: Revert this please

**Comment by feigenmann** [comment]
Obsoleted by #644
